stdlib/2and3/rlcompleter.pyi

```diff
@@ -10,4 +10,5 @@ else:
 
 
 class Completer:
+    def __init__(self, namespace: Optional[dict] = ...) -> None: ...
```

Comment:
Can you add type arguments to the dict? (I assume something like `Dict[str, str]` but I haven't checked.)